### PR TITLE
feat:quiz questions, rules

### PR DESCRIPTION
**Pull request by Naveenchandar**
* Rules section
* Quiz questions

screencapture-localhost-3000-rules-2022-04-03-16_07_12
screencapture-localhost-3000-quiz-2022-04-03-16_07_35

### CODE REVIEW

File: src/Components/Routes/index.js [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+            <Route path="/result" element={<Result />} />
+            <Route path="/rules" element={<Rules />} />
             <Route
                 path="*"
                 element={
-                    <main>
+                    <main
+                        style={{
+                            padding: "1rem",
+                            // backgroundColor: "var(--background-color)",
```
Comment (Aditi0505) [inline src/Components/Routes/index.js:17]: Please avoid inline styles. Create a class for the same and use className here.

File: src/Context/dataContext.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  useEffect(() => {
+    (async function fetchQuestions() {
+      try {
+        const { data } = await axios.get(
+          `https://opentdb.com/api.php?amount=10&category=${category.id}&difficulty=medium&type=multiple`
+        );
+        setData(data.results);
+      } catch (error) {
+        console.log(error);
```
Comment (Aditi0505) [inline src/Context/dataContext.js:19]: Display error on UI.

**Review by Aditi0505 — APPROVED** [review]
- Good use of context and reducers
- Good naming convention followed
- Good folder structure

File: src/Context/resultContext.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React, { useReducer, createContext, useContext } from "react";
+import { resultReducer } from "../Reducer/result-reducer";
```
Comment (anupkgurung) [inline src/Context/resultContext.js:2]: use index file to export(barrel export)

**Review by anupkgurung — APPROVED** [review]
Code is clean.
merge could be broken down into smaller and short commits.

**Pull request merged**